input/pagecontent/5_testplan.md

```diff
@@ -1,8 +1,69 @@
+<div markdown="1" class="stu-note">
 
-For FHIR Evaluation use [EVSClient](https://gazelle.ihe.net/content/evsfhirvalidation) or [HL7 FHIR Validator](https://confluence.hl7.org/display/FHIR/Using+the+FHIR+Validator) using the npm package [IHE.ITI.PIXm](package.tgz). 
+This Test Plan page is a prototype.   We expect the maturity of the content will improve over time.  For now, we summarize high level testing scope and available tools. Comments are welcome.
+</div>
 
-For PIXm Consumers wanting to query a [PIXm Manager simulator in Gazelle Patient Manager](https://gazelle.ihe.net/content/pmpixquery-patientidentityconsumer)
+## Introduction
 
-For PIXm Managers wanting to use a [PIXm Consumer simulator in Gazelle Patient Manager]( https://gazelle.ihe.net/content/pmpixquery-pixmanager)
+PIXm specifies a query transaction between two actors.  The transaction between actors specify semantics of the data exchanged.  The PDQm test plan focuses on these semantics and on the expected actions on the server-side actor.
```

Comment:
Instead of 'The PDQm test plan' the PIXm test plan?

Reply:
should we mention also the feed transaction? PIXm specifies a query and a feed transaction between to actors.

Reply:
yes PIXm, not PDQm (copy/paste error)

yes, I missed adding the feed.   I can fix both of these.